hack/tools/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/golangci/golangci-lint v1.43.0
 	github.com/jteeuwen/go-bindata v3.0.7+incompatible
-	k8s.io/code-generator v0.22.2
+	k8s.io/code-generator v0.23.0
```

Comment:
should not this be `v0.23.0-alpha.4` as per linked comment in PR description?

Reply:
The release of 1.23.0 is out now, so I think that's okay?

Reply:
Okay